src/classes/TDTM_DMLgt200_TEST.cls

```diff
@@ -69,39 +69,32 @@ private class TDTM_DMLgt200_TEST {
         UTIL_CustomSettings_API.getSettingsForTests(new Hierarchy_Settings__c(Account_Processor__c = UTIL_Describe_API.getAdminAccRecTypeID()));
 
         String newContactMailingStreet = '123 Elm St';
-
-		List<Contact> testContacts = new List<Contact>();
-
+		List<Contact> testContacts = UTIL_UnitTestData_TEST.getMultipleTestContacts(inputSize);
 		for (Integer i = 0; i < inputSize; i++) {
-        	Contact con = new Contact(
-            FirstName='Test',
-            LastName='Contact_forTests',
-            MailingStreet = newContactMailingStreet,
-            WorkEmail__c = 'junk@test.net',
-            Preferred_Email__c = 'Work',
-            WorkPhone__c = '206-777-8888',
-            PreferredPhone__c = 'Work');
-            testContacts.add(con);
+            testContacts[i].OtherCity = null;
+            testContacts[i].MailingStreet = newContactMailingStreet;
         }
         insert testContacts;
 
-        List<Contact> insertedContacts = [Select FirstName, LastName, AccountId, Account.Name,Account.Primary_Contact__c,MailingStreet,
-        									   Account.BillingStreet 
-        							    	from Contact];
+        List<Contact> insertedContacts = [SELECT FirstName, LastName, AccountId, Account.Name, Account.Primary_Contact__c, WorkEmail__c, WorkPhone__c, Email, Phone
+        							    	FROM Contact];
 
-        //one contact should have been created
+        //many contacts should have been created
         system.assertEquals(inputSize, insertedContacts.size());
 
         //relationship should be bi-directional
         for (Contact con : insertedContacts) {
-	        system.assertEquals(con.id, con.Account.Primary_Contact__c);   
+	        system.assertEquals(con.id, con.Account.Primary_Contact__c);
+            system.assertEquals(con.WorkEmail__c, con.Email);
```

Comment:
These two lines are the two new pieces of code that really test that the bug is fixed for > 200 records. When inserting > 200 records, these assertions were false after record 200.